pom.xml

```diff
@@ -262,6 +263,12 @@
             <version>${powermock.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.freemarker</groupId>
+            <artifactId>freemarker</artifactId>
+            <version>2.3.30</version>
```

Comment:
Let's move the dependency version to the properties section similar to the other dependencies

Reply:
Updated in 5f317f92c850f22d5567b29bc5e483a84413d44e